### PR TITLE
[Frontend|MXNet] SwapAxis operator support

### PR DESCRIPTION
**Pull request by maheshambule**
Reference:
https://beta.mxnet.io/api/ndarray/_autogen/mxnet.ndarray.swapaxes.html

@kazum, @kevinthesun, @alexgl-github Please help in reviewing this.

### CODE REVIEW

File: python/tvm/relay/frontend/nnvm_common.py [hunk 1]
```diff
@@ -108,6 +108,17 @@ def _transpose(inputs, attrs):
     return _op.transpose(inputs[0], axes=axes)
 
 
+def _swap_axis(inputs, attrs):
```
Comment (kazum) [inline python/tvm/relay/frontend/nnvm_common.py:111]: I think it's better to define this in mxnet.py as _mx_swapaxis().

Comment (maheshambule) [reply python/tvm/relay/frontend/nnvm_common.py]: done.

**Comment by maheshambule** [comment]
@kazum Thanks. I have made the changes as per the comment.

**Comment by kevinthesun** [comment]
Thanks @maheshambule @kazum

**Pull request merged**